examples/layouts/toolbar.html

```diff
@@ -5,25 +5,91 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Main Layout CDN</title>
+  <title>Generic Layout</title>
   <link rel="stylesheet" href="../../packages/styles/dist/airship.css">
+  <script src="../../packages/components/dist/airship.js"></script>
+
+  <!-- INCLUDE TYPOGRAFY -->
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
```

Comment:
Should we import Roboto via @import? à la `@import url('https://fonts.googleapis.com/css?family=Roboto+Mono');`

Just suggesting. I don't know the best way to do it but I'd appreciate to free the user to include the tag to use Airship as it's thought to look.

Reply:
Yes, I am not sure about it either.

I talked it the other day with @IagoLast but we didn't reach any conclusion. I'd say that I would include it in CSS and that's it, but 🤷‍♂️ . We can talk about it :)